k8s/ithc/common/ccd/definition-store-api.yaml

```diff
@@ -19,9 +19,22 @@ spec:
   values:
     java:
       replicas: 2
+      keyVaults:
+        ccd:
+          secrets:
+            - definition-store-api-POSTGRES-USER-V11
+            - definition-store-api-POSTGRES-PASS-V11
+            - ccd-definition-s2s-secret
+            - ccd-ELASTIC-SEARCH-URL
+            - ccd-ELASTIC-SEARCH-PASSWORD
+            - storage-account-primary-connection-string
+            - AppInsightsInstrumentationKey
       useInterpodAntiAffinity: true
       image: hmctspublic.azurecr.io/ccd/definition-store-api:pr-974-398e4709
       environment:
+        DEFINITION_STORE_DB_HOST: ccd-definition-store-api-postgres-db-v11-ithc.postgres.database.azure.com
+        DEFINITION_STORE_DB_NAME: ccd_definition_store
+        DEFINITION_STORE_DB_USERNAME: ccd@ccd-definition-store-api-postgres-db-v11-ithc
```

Comment:
One min, check in azure if it is the generated DEFINITION_STORE_DB_NAME VALUE for version 11.  "ccd_definition_store" . Just to check

Reply:
copied these from azure